### ISSUE TITLE
NAT config fails without default network

### ISSUE THREAD

**Issue by aarontp** [forseti-security/osdfir-infrastructure#32]
The current deploy.sh script tries to create a NAT config, but if the default network is not available it complains.  We should either create a default network, or potentially just not create a NAT config if the default network does not exist.

```
ERROR: (gcloud.compute.networks.subnets.update) Could not fetch resource:
 - The resource 'projects/<redacted>/regions/<redacted>/subnetworks/default' was not found
```

**Comment by aarontp** [comment]
FYI @hacktobeer

**Comment by aarontp** [comment]
Also an issue for "legacy" networks"
```
ERROR: (gcloud.compute.routers.nats.create) HTTPError 400: Invalid value for field 'resource.network': 'https://www.googleapis.com/compute/v1/projects/<redacted>/global/networks/default'. Nat service is not available for legacy network.
```

**Comment by hacktobeer** [comment]
NAT is indeed not supported with legacy networks. https://cloud.google.com/vpc/docs/using-legacy

**Comment by hacktobeer** [comment]
"You cannot convert a legacy network into a VPC network. If you need the features of a VPC network, you have to rebuild your configuration in a VPC network, migrate or recreate your VMs, then point DNS or your services at the new setup."

That sounds horrible, let's see what we can do.

**Comment by aarontp** [comment]
Fixed by #33
